runtime/src/main/jni/MetadataNode.cpp

```diff
@@ -807,6 +814,7 @@ void MetadataNode::SetInstanceMetadata(Isolate* isolate, Local<Object> object, M
 }
 
 void MetadataNode::ExtendedClassConstructorCallback(const v8::FunctionCallbackInfo<v8::Value>& info) {
+    TNSPERF();
```

Comment:
What's the difference between using the TNSPERF macro and tns::instrumentation::Frame constructor? Can we make it consistent across the runtime code, and use just one of the two?